src/libraries/System.Runtime.Extensions/tests/System/StringComparer.cs

```diff
@@ -86,8 +86,10 @@ public static IEnumerable<object[]> UpperLowerCasing_TestData()
             //                          lower                upper          Culture
             yield return new object[] { "abcd",             "ABCD",         "en-US" };
             yield return new object[] { "latin i",          "LATIN I",      "en-US" };
-            yield return new object[] { "turky \u0131",     "TURKY I",      "tr-TR" };
-            yield return new object[] { "turky i",          "TURKY \u0130", "tr-TR" };
+            if (!PlatformDetection.IsBrowser){
```

Comment:
Can we wait to exclude?  It would be easy to miss when we add some sort of ICU feature detection.

Reply:
Done.